tools/base/BUILD

```diff
@@ -0,0 +1,46 @@
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
+load("//bazel:envoy_build_system.bzl", "envoy_package")
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+py_library(
+    name = "checker",
+    srcs = ["checker.py"],
+)
+
+py_library(
+    name = "utils",
+    srcs = ["utils.py"],
+)
+
+py_library(
+    name = "test_utils",
+    srcs = ["tests/test_utils.py"],
+)
+
+# do this better
```

Comment:
not sure how to do this better - i tried playing with a `genrule` but the python didnt seem to work

it found the tests but the test wasnt able to see the target - probs a pythonpath issue - but my hacky attempts did not fix